Resources/Prototypes/DeltaV/Catalog/Fills/Lockers/chiefjustice.yml

```
@@ -0,0 +1,15 @@
+- type: entity
+  parent: LockerChiefJustice
+  id: LockerChiefJusticeFilled
+  name: chief justice's locker
+  components:
+  - type: StorageFill
+    contents:
+    - id: ClothingHeadsetCJ
+    - id: ClothingNeckCloakCJ
+    - id: BoxPDAJustice
+    - id: BoxEncryptionKeyJustice
+    - id: CJIDCard
```

CJ? From hit video-gameGTA San Andreas?

```suggestion
    - id: ChiefJusticeIDCard
```